src/bin/cachepot-dist/main.rs

```diff
@@ -90,7 +89,7 @@ struct GenerateJwtHS256ServerToken {
 
     /// Generate a key for the specified server
     #[structopt(long, value_name = "SERVER_ADDR", required_unless = "secret_key")]
-    server: SocketAddr,
+    server: String,
```

Comment:
`#[structopt(parse(from_os_str))]` should avoid the deferring parsing I'd assume?

Reply:
right